charts/stable/dnsmadeeasy-webhook/templates/common.tpl

```diff
@@ -62,7 +62,7 @@ probes:
 service:
   main:
     ports:
```

Comment:
This needs to be
```yaml
    ports:
      http:
        enabled: false
      https:
        enabled: true
        targetPort: 4443
        port: 443
```
Otherwise the `https` port will not be enabled and the `http` port spec will be invalid.